### PR TITLE
Replace yaml with json

### PR DESCRIPTION
**Pull request by FrauR**
Replace the 'feature.yml' by a 'feature.json' to improve inheritance behavior between features.

### CODE REVIEW

**Comment by FrauR** [comment]
@michaelengland can you have a look please.

**Comment by FrauR** [comment]
@michaelengland

File: lib/flip_the_switch/reader/features.rb [hunk 1]
```diff
@@ -1,47 +1,170 @@
-require 'yaml'
+require 'json'
+require 'json-schema'
 
 module FlipTheSwitch
   module Reader
     class Features
+      INHERITANCE_INDICATOR = 'inherits_from'
```
Comment (michaelengland) [inline lib/flip_the_switch/reader/features.rb]: I think this is now to-go. Forgot that one :)

**Comment by michaelengland** [comment]
Looking great. Nice work! :+1:

**Pull request merged**